### ISSUE TITLE
CMSSW_12_4_0_pre2 profiles

### ISSUE THREAD

**Issue by jpata** [jpata/cms-reco-profiling-web#53]
The new profiles for CMSSW_12_4_0_pre2 are ready. Please propagate them through the `release-analyze-reco-profiling` step, and let me know in case of any issues.

```
[jpata@lxplus797 cms-reco-profiling]$ ls /eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_4_0_pre2/slc7_amd64_gcc10/*
/eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_4_0_pre2/slc7_amd64_gcc10/11834.21:
cmdLog_profiling.sh         step3_igprofCPU.399.gz    step4_circles.json          step4_JobReport.xml         step5_igprofMEM.1.gz
step1_JobReport.xml         step3_igprofCPU.gz        step4_FastTimerService.log  step4.log                   step5_igprofMEM.200.gz
step1.log                   step3_igprof_cpu.txt      step4_igprofCPU.1.gz        step4.root                  step5_igprofMEM.399.gz
step1.root                  step3_igprofMEM.1.gz      step4_igprofCPU.200.gz      step4_TimeMemoryInfo.log    step5_igprof_mem.txt
step2_JobReport.xml         step3_igprofMEM.200.gz    step4_igprofCPU.399.gz      step5_circles.json          step5_JobReport.xml
step2.log                   step3_igprofMEM.399.gz    step4_igprofCPU.gz          step5_FastTimerService.log  step5.log
step2.root                  step3_igprof_mem.txt      step4_igprof_cpu.txt        step5_igprofCPU.1.gz        step5.root
step3_circles.json          step3_JobReport.xml       step4_igprofMEM.1.gz        step5_igprofCPU.200.gz      step5_TimeMemoryInfo.log
step3_FastTimerService.log  step3.log                 step4_igprofMEM.200.gz      step5_igprofCPU.399.gz
step3_igprofCPU.1.gz        step3.root                step4_igprofMEM.399.gz      step5_igprofCPU.gz
step3_igprofCPU.200.gz      step3_TimeMemoryInfo.log  step4_igprof_mem.txt        step5_igprof_cpu.txt

/eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_4_0_pre2/slc7_amd64_gcc10/35234.21:
cmdLog_profiling.sh  step3_FastTimerService.log  step3_igprofMEM.99.gz       step4_igprofCPU.1.gz   step4_igprof_mem.txt
step1_JobReport.xml  step3_igprofCPU.1.gz        step3_igprof_mem.txt        step4_igprofCPU.50.gz  step4_JobReport.xml
step1.log            step3_igprofCPU.50.gz       step3_JobReport.xml         step4_igprofCPU.99.gz  step4.log
step1.root           step3_igprofCPU.99.gz       step3.log                   step4_igprofCPU.gz     step4.root
step2_JobReport.xml  step3_igprofCPU.gz          step3.root                  step4_igprof_cpu.txt   step4_TimeMemoryInfo.log
step2.log            step3_igprof_cpu.txt        step3_TimeMemoryInfo.log    step4_igprofMEM.1.gz
step2.root           step3_igprofMEM.1.gz        step4_circles.json          step4_igprofMEM.50.gz
step3_circles.json   step3_igprofMEM.50.gz       step4_FastTimerService.log  step4_igprofMEM.99.gz

/eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_4_0_pre2/slc7_amd64_gcc10/136.889:
cmdLog_profiling.sh  step2.log   step3_JobReport.xml  step3.root                step4_JobReport.xml
step2_JobReport.xml  step2.root  step3.log            step3_TimeMemoryInfo.log  step4.log
```

@xoqhdgh1002 @yeopjjang

**Comment by jpata** [comment]
A kind ping on this. Now with the automatic workflow, it should take no more than a few days to update the web page once the profiles are available. Let me know if there's something blocking it.

**Comment by jpata** [comment]
solved by #55
